.github/workflows/upmerge_pr.yaml

```diff
@@ -21,16 +21,10 @@ jobs:
             matrix:
                 include:
                     -
-                        base_branch: "2.1"
+                        base_branch: "1.0"
                         target_branch: "2.2"
                     -
                         base_branch: "2.2"
-                        target_branch: "3.0"
-                    -
-                        base_branch: "3.0"
-                        target_branch: "3.1"
-                    -
-                        base_branch: "3.1"
```

Comment:
Since we have not yet released 3.2, the currently supported version is 3.1. I would leave 2.2 -> 3.1 and 3.1 -> 3.2, at least for now.